engine/src/main/java/com/datatorrent/stram/webapp/OperatorDiscoverer.java

```diff
@@ -167,17 +179,21 @@ else if (qName.equalsIgnoreCase("method")) {
       }
     }
 
-    private void addTagToMethod(Map<String, MethodInfo> methods, String tagText, boolean isDescription)
+    private void addTagToMethod(Map<String, MethodInfo> methods, String tagText, MethodTagType tagType)
```

Comment:
Since you have MethodTagType enum, you can put it in the enum method so you don't need those if else and the if else above